src/usePromiseWithInterval.ts

```diff
@@ -7,6 +7,11 @@ export const usePromiseWithInterval = <T, Args extends any[], E = string>(
     interval: number,
 ): UsePromiseWithInterval<T, E, Args> => {
     const [result, load, reset] = usePromise<T, Args, E>(loaderFn);
+    const [retries, setRetries] = useState<number>(0);
```

Comment:
minor thing, but it's about naming ;)
if the first load increments that counter, and the first load is not a `_retry_` per se, I'd do either of two things:
- keep the name `retries` and not increment the counter after first load
- rename to `tryCount` and keep incrementation logic as is

Make sure to add that description to README :)

Reply:
Renamed to `tryCount`